tephra-hbase-compat-1.1-base/src/main/java/org/apache/tephra/hbase/txprune/InvalidListPruningDebugTool.java

```diff
@@ -165,10 +165,10 @@ public void destroy() throws IOException {
    */
   @Override
   @SuppressWarnings("WeakerAccess")
-  public SortedSet<RegionPruneInfoPretty> getIdleRegions(Integer numRegions, String time) throws IOException {
+  public List<RegionPruneInfoPretty> getIdleRegions(Integer numRegions, String time) throws IOException {
```

Comment:
so now the result won't be sorted? 

Another option is to keep the sorted set, but make the region name part of the comparable. So that if two have the same prune bound, they still have different sort keys but end up adjacent to each other in the result. That would be a sorted result.

Reply:
👍